conf/rubocop_rails.yml

```diff
@@ -1,9 +1,19 @@
 inherit_from:
   - ../conf/rubocop.yml
 
+inherit_mode:
+  merge:
+    - Exclude
```

Comment:
Is it possible to have this apply to only specific rules, so we don't suddenly change this across the board?

Reply:
Awesome!!